client/scss/components/_button.scss

```
@@ -355,4 +355,17 @@
       }
     }
   }
+
+  // Ensure visual consistency between button and a elements in WHCM
+
+  @media (forced-colors: active) {
```

```suggestion
  @media (forced-colors: $media-forced-colours) {
```

I will fix this up but we try to use the variables here for any new usage.